### DISCUSSION TITLE
Can we solve time-dependent PDEs with exogenous input?

### DISCUSSION THREAD

**Discussion by LeXu1327** [lululxvi/deepxde#882] [Q&A]
![ask](https://user-images.githubusercontent.com/112776354/188283426-9ca54f3c-cf17-4f92-b500-c9c577817a09.png)

**Comment by praksharma** [comment]
Yes, people call it transfer learning. You slightly update the last layer of the neural network with very small learning rate. 

Check [this](https://github.com/lululxvi/deepxde/issues/195) for DeepXDE.
